packages/components/src/custom-select-control/index.js

```diff
@@ -56,6 +56,7 @@ export default function CustomSelectControl( {
 	className,
 	hideLabelFromVision,
 	label,
+	describedBy,
```

Comment:
I guess we should document the new property on packages/components/src/custom-select-control/README.md.

Reply:
Done. Do you think it is clear enough?

Reply:
Made another update to usage.